src/Defaults.js

```diff
@@ -2,5 +2,6 @@ export default {
 	baseUrl: 'https://api.particle.io',
 	clientSecret: 'particle-api',
 	clientId: 'particle-api',
-	tokenDuration: 7776000 // 90 days
+	tokenDuration: 7776000, // 90 days
+	timeout: 60000
```

Comment:
Is this value going to be used anywhere? It sets `particle.timeout = 60000` but `this.timeout` is not used in the `Particle` class. If it's unused let's remove this line.